packages/kbn-es/src/utils/docker.test.ts

```diff
@@ -438,7 +438,6 @@ describe('resolveEsArgs()', () => {
       kibanaUrl: 'https://localhost:5601/',
     });
 
-    expect(esArgs).toHaveLength(26);
```

Comment:
__note:__ the Jest snapshot already verifies that we have the expected number of args.